.luarc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://raw.githubusercontent.com/sumneko/vscode-lua/master/setting/schema.json",
+    "Lua.diagnostics.globals": [
+        "vim",
+        "nvchad"
```

Comment:
without this, we get a warning that says `nvchad` is not in the global environment.

Reply:
@ahhshm ig its not needed, u add it in our lspconfig's sumneko section . just like we added "vim" there

![image](https://user-images.githubusercontent.com/59060246/167650021-3f0f0bb5-3ff0-40dc-a3d0-414c3864432e.png)

Reply:
Ok. i will remove it